docs/root/configuration/network_filters/thrift_proxy_filter.rst

```diff
@@ -0,0 +1,16 @@
+.. _config_network_filters_thrift_proxy:
+
+Thrift proxy
+============
+
+* The Thrift proxy is not supported in the v1 API.
```

Comment:
I'd drop any mention of v1 API; it's about to disappear at the next release cycle, so one fewer places to clean up.